packages/eslint-config-react/package.json

```diff
@@ -21,5 +21,9 @@
     "babel-eslint": "^10.0.3",
     "eslint-plugin-react": "^7.17.0",
     "eslint-plugin-react-hooks": "^2.5.0"
+  },
+  "peerDependencies": {
```

Comment:
Would it make sense to move all peerDependencies to devDependencies? Since we will force people to install them themselves we might as well not ship with the dependencies in our package.

Reply:
I suppose you mean the other way around: dev to peer. But ideally you require both: dev to test your plugin and peer to warn consumers they should have it installed.

Reply:
No, what I meant is that these 2 `peerDependencies` are listed also as `dependencies`, which does not make sense, If we require people to install the dependencies themselves, then we shouldn't ship them with the package by making them `devDependencies`